components/shell/mosh/Makefile

```diff
@@ -36,29 +37,26 @@ COMPONENT_LICENSE=      GPLv3+
 COMPONENT_LICENSE_FILE=	$(COMPONENT_NAME).license 
 COMPONENT_SUMMARY=	Mosh (mobile shell) - Remote terminal application that allows roaming, supports intermittent connectivity, and provides intelligent local echo and line editing of user keystrokes (replacement for SSH).
 
-include $(WS_MAKE_RULES)/prep.mk
-include $(WS_MAKE_RULES)/configure.mk
-include $(WS_MAKE_RULES)/ips.mk
+TEST_TARGET=		$(NO_TESTS)
+include $(WS_MAKE_RULES)/common.mk
 
-CONFIGURE_BINDIR.64= $(CONFIGURE_BINDIR.32)
+LDFLAGS += -lutempter
```

Comment:
should we just configure mosh with `--with-utempter`?

Reply:
That would also work, the only difference I see here is with gmake REQUIRED_PACKAGES:
> REQUIRED_PACKAGES += $(GCC_RUNTIME_PKG)
71d72
< REQUIRED_PACKAGES += system/library/gcc-4-runtime

("LDFLAGS += -lutempter" version determines the first dependency, "CONFIGURE_OPTIONS+= --with-utempter" the second one.)